db/migrate/20170316160949_add_auth_bypass_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddAuthBypassId < ActiveRecord::Migration[5.0]
+  def change
+    add_column :access_limits, :auth_bypass_ids, :json, null: false, default: []
```

Comment:
is this migration quick? adding default ones can sometimes be quite painful.

Also will this cause any problems for existing data in fact_check_ids?